android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -175,24 +188,39 @@ FUNC(jboolean, isAvifImage, jobject encoded, int length) {
   return avifPeekCompatibleFileType(&avif);
 }
 
+#define CHECK_EXCEPTION(ret) \
+  if (JniExceptionCheck(env)) return ret
+
+#define FIND_CLASS(var, class_name, ret)         \
+  const jclass var = env->FindClass(class_name); \
+  CHECK_EXCEPTION(ret);                          \
+  if (var == nullptr) return ret
```

Comment:
The null pointer checks of the return values of `env->FindClass()` and `env->GetFieldID()` are also new in this pull request, right? It would be good to mention this in the commit message.

Is it a bug that we were not checking their return values before?

Reply:
> The null pointer checks of the return values of `env->FindClass()` and `env->GetFieldID()` are also new in this pull request, right? It would be good to mention this in the commit message.
> 

Updated the commit message.

> Is it a bug that we were not checking their return values before?

Yes, these functions could fail in one of two ways: throw an exception or return nullptr. This PR handles both of those cases.